app/src/main/java/com/example/news/model/CachedData.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.news.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+
+@Entity(tableName = "News")
+data class CachedData(
+    @PrimaryKey(autoGenerate = true)
+    val id:Int,
+    val cachedData: List<Article>
```

Comment:
create a new article class for the database entity. @RadwaMohamed29

Reply:
fixed